packages/devtools_app/lib/src/app.dart

```diff
@@ -266,7 +266,7 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
       appSizePageId: (_, __, ___) {
         return DevToolsScaffold.withChild(
           key: const Key('appsize'),
-          child: _providedControllers(
+          _providedControllers(
```

Comment:
in the case of calling a method, I don't know whether `_providedControllers` returns a list or a single widget, so I don't know if this is a single child or a list of children that I can add to.